lib/v2/blogs/love-the-problem/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    '/blogs/love-the-problem': {
+        _name: 'Love the Problem',
+        '.': [
+            {
+                title: 'Love the Problem',
```

Comment:
I think `Ash Maurya's blog` is better.

Reply:
I was considering this. Should we change the name everywhere?

Reply:
route: /blogs/ash-maurya ?